packages/fuels-programs/src/contract.rs

```diff
@@ -906,7 +915,9 @@ mod test {
     use super::*;
 
     #[tokio::test]
-    #[should_panic(expected = "called `Result::unwrap()` on an `Err` value: InvalidData(\"json\")")]
+    #[should_panic(
+        expected = "called `Result::unwrap()` on an `Err` value: InvalidData(\"The file extension 'json' is not recognized. Did you mean '.bin'?\")"
+    )]
```

Comment:
`should_panic` will check if the provided string is part of the error message. IMO this makes it a bit nicer...
```suggestion
    #[should_panic(expected = "The file extension 'json' is not recognized. Did you mean '.bin'?")]
    )]
```